### ISSUE TITLE
check with v2022.10.12 detects 52365 but v2022.12.19 does not

### ISSUE THREAD

**Issue by dmtucker** [pypa/pipenv#5559]
``` console
$ pipenv check
Checking PEP 508 requirements...
Passed!
Checking installed package safety...
52365: certifi <2022.12.07 resolved (2022.9.24 installed)!
Certifi 2022.12.07 includes a fix for CVE-2022-23491: Certifi 2022.12.07 removes root certificates from "TrustCor" from the root store. These are in the process of being removed from Mozilla's trust store. TrustCor's root certificates are being removed pursuant to an investigation prompted by media reporting that TrustCor's ownership also operated a business that produced spyware. Conclusions of Mozilla's investigation can be found in the linked google group discussion.
https://github.com/certifi/python-certifi/security/advisories/GHSA-43fp-rhv2-5gv8
https://groups.google.com/a/mozilla.org/g/dev-security-policy/c/oxX69KFvsm4/m/yLohoVqtCgAJ

$ pipenv --version
pipenv, version 2022.10.12
```
``` console
$ pipx upgrade-all
upgraded package pipenv from 2022.10.12 to 2022.12.19 (location: ...)
```
``` console
$ pipenv check
Checking PEP 508 requirements...
Passed!
Checking installed packages for vulnerabilities...
+================================================================+

                               /$$$$$$            /$$
                              /$$__  $$          | $$
           /$$$$$$$  /$$$$$$ | $$  \__//$$$$$$  /$$$$$$   /$$   /$$
          /$$_____/ |____  $$| $$$$   /$$__  $$|_  $$_/  | $$  | $$
         |  $$$$$$   /$$$$$$$| $$_/  | $$$$$$$$  | $$    | $$  | $$
          \____  $$ /$$__  $$| $$    | $$_____/  | $$ /$$| $$  | $$
          /$$$$$$$/|  $$$$$$$| $$    |  $$$$$$$  |  $$$$/|  $$$$$$$
         |_______/  \_______/|__/     \_______/   \___/   \____  $$
                                                          /$$  | $$
                                                         |  $$$$$$/
  by pyup.io                                              \______/

+================================================================+

 REPORT 

  Safety v2.3.2 is scanning for
    Vulnerabilities...
  Scanning dependencies in your files:

  -> /tmp/searents-IxmFmDqUs1z0p52e_requirements.txt

  Found and scanned 12 packages
  Timestamp 2023-01-01 18:48:02
  0 vulnerabilities found
  0 vulnerabilities ignored
+================================================================+

 No known security vulnerabilities found. 

+================================================================+
$ pipenv graph | grep certifi
    - certifi [required: >=2017.4.17, installed: 2022.9.24
$ pipenv --version
pipenv, version 2022.12.19
```

**Comment by matteius** [comment]
Hey @yeisonvargasf I hope you had a good end to 2022 -- it looks like this issue is related to the upgrade to safety 2, could you comment on this when you get a chance?

**Comment by yeisonvargasf** [comment]
Hi @matteius, thank you. I hope all is going great with you.

> it looks like this issue is related to the upgrade to safety 2, could you comment on this when you get a chance?

We are addressing the issue, and soon there should no longer be happening. I'll make sure to confirm here when this is solved.

**Comment by dmtucker** [comment]
It seems to be the other way around now
``` console
$ venv/bin/pipenv --version
pipenv, version 2022.10.12
$ venv/bin/pipenv check
Checking PEP 508 requirements...
Passed!
Checking installed package safety...
All good!
$ venv/bin/pip install -U pipenv
Requirement already satisfied: pipenv in ./venv/lib/python3.10/site-packages (2022.10.12)
Collecting pipenv
  Using cached pipenv-2022.12.19-py3-none-any.whl (2.9 MB)
Requirement already satisfied: virtualenv-clone>=0.2.5 in ./venv/lib/python3.10/site-packages (from pipenv) (0.5.7)
Requirement already satisfied: certifi in ./venv/lib/python3.10/site-packages (from pipenv) (2022.12.7)
Requirement already satisfied: setuptools>=36.2.1 in ./venv/lib/python3.10/site-packages (from pipenv) (66.1.1)
Requirement already satisfied: virtualenv in ./venv/lib/python3.10/site-packages (from pipenv) (20.17.1)
Requirement already satisfied: distlib<1,>=0.3.6 in ./venv/lib/python3.10/site-packages (from virtualenv->pipenv) (0.3.6)
Requirement already satisfied: platformdirs<3,>=2.4 in ./venv/lib/python3.10/site-packages (from virtualenv->pipenv) (2.6.2)
Requirement already satisfied: filelock<4,>=3.4.1 in ./venv/lib/python3.10/site-packages (from virtualenv->pipenv) (3.9.0)
Installing collected packages: pipenv
  Attempting uninstall: pipenv
    Found existing installation: pipenv 2022.10.12
    Uninstalling pipenv-2022.10.12:
      Successfully uninstalled pipenv-2022.10.12
Successfully installed pipenv-2022.12.19
$ venv/bin/pipenv check
Checking PEP 508 requirements...
Passed!
Checking installed packages for vulnerabilities...
+====================================================================================================================================+

                               /$$$$$$            /$$
                              /$$__  $$          | $$
           /$$$$$$$  /$$$$$$ | $$  \__//$$$$$$  /$$$$$$   /$$   /$$
          /$$_____/ |____  $$| $$$$   /$$__  $$|_  $$_/  | $$  | $$
         |  $$$$$$   /$$$$$$$| $$_/  | $$$$$$$$  | $$    | $$  | $$
          \____  $$ /$$__  $$| $$    | $$_____/  | $$ /$$| $$  | $$
          /$$$$$$$/|  $$$$$$$| $$    |  $$$$$$$  |  $$$$/|  $$$$$$$
         |_______/  \_______/|__/     \_______/   \___/   \____  $$
                                                          /$$  | $$
                                                         |  $$$$$$/
  by pyup.io                                              \______/

+====================================================================================================================================+

 REPORT 

  Safety v2.3.2 is scanning for Vulnerabilities...
  Scanning dependencies in your files:

  -> /tmp/tmp.suT66hVcJ5-rC_iRD1op6bm0iil_requirements.txt

  Found and scanned 4 packages
  Timestamp 2023-01-27 09:46:01
  1 vulnerability found
  0 vulnerabilities ignored

+====================================================================================================================================+
 VULNERABILITIES FOUND 
+====================================================================================================================================+

-> Vulnerability found in certifi version 2022.9.24
   Vulnerability ID: 52365
   Affected spec: <2022.12.07
   ADVISORY: Certifi 2022.12.07 includes a fix for CVE-2022-23491: Certifi 2022.12.07 removes root certificates from
   "TrustCor" from the root store. These are in the process of being removed from Mozilla's trust store. TrustCor's root...
   CVE-2022-23491
   For more information, please visit https://pyup.io/v/52365/742

 Scan was completed. 1 vulnerability was found. 

+====================================================================================================================================+
   REMEDIATIONS

  1 vulnerability was found in 1 package. For detailed remediation & fix recommendations, upgrade to a commercial license. 

+====================================================================================================================================+
```
That's fine with me if that's expected :+1:

**Comment by matteius** [comment]
@dmtucker The default behavior of `pipenv check` changed in this release to check against the packages in the Pipfile.lock by default, but if you want the old behavior pass `--use-installed` to the command.   I think the check command is working well, but we can re-open this if there needs to be more done.
